src/main/java/org/terracotta/build/plugins/AngelaPlugin.java

```diff
@@ -53,13 +57,14 @@ public void apply(Project project) {
     ConfigurationContainerInternal configurations = (ConfigurationContainerInternal) project.getConfigurations();
 
     Provider<Directory> angelaDir = project.getLayout().getBuildDirectory().dir("angela");
+    Provider<Directory> fakeHostDir = project.getLayout().getBuildDirectory();
     Provider<Directory> customKitDir = angelaDir.map(d -> d.dir("custom-tc-db-kit"));
 
     Configuration angela = configurations.bucket(FRAMEWORK_CONFIGURATION_NAME);
     angela.defaultDependencies(defaultDeps -> {
       defaultDeps.add(project.getDependencyFactory().create("org.terracotta", "angela", "[3,)"));
     });
-
+    AngelaPluginExtension angelaPluginExtension = project.getExtensions().create("customHostsConfig", AngelaPluginExtension.class);
```

Comment:
is it possible to call this extension "angela" instead ?

Reply:
u want this to be angela instead of customHostsConfig ? Or u want AngelaPluginExtension to be renamed as AngelaExtension ?